_overview/overview-of-spinnaker.md

```diff
@@ -4,39 +4,63 @@ title: Overview of Armory Spinnaker
+  - **Rolling Blue/Green (Red/Black)**:  Similar to Blue/Green, but traffic is gradually shifted from the older deployment to the new one.
+  - **Highlander**:  Similar to Blue/Green, except the old deployment is destroyed once traffic is shifted.
+  - **Canary**: This consists of three instances: the current production instance, a baseline instance (a smaller clone of production), and a canary intance with the new deployment. The production instance handles most of the load while the baseline and canary each receive a smaller amount. After a predetermined amount of time, performance of the baseline and canary are compared. Whether the a deployment becomes the new production build depends on canary analysis that can be automated or manual.
+- **Automated canary analysis** through Kayenta, a canary analysis tool that is integrated with Spinnaker. Without manual intervention, Kayenta can determine if a canary deployment should be pushed to production.
+- **Multi-cloud deployments** to avoid lock-in and allow you to optimize for things like cost, latency, and geographic distribution.
 
 ## Why is it relevant?
 
-Today’s world revolves around software and services working reliably and continuously - the internet is increasingly accessible from all time-zones and having business services experience downtime is becoming increasingly costly. Spinnaker is an advanced tool for cloud deployments that not only encourages businesses to move to the cloud (an expected migration) but makes it easier for them to adopt the cloud’s advantages, such as immutable infrastructure and multi-cloud deployments.
+Shipping changes more frequently allows developers to gather real user feedback sooner, enabling them to iterate and build based on actual input from customers. Additionally, Spinnaker abstracts away much of the cloud configuration details, giving developers more time to focus on meaningful tasks instead of infrastructure details.
 
 
 ## What Armory Spinnaker Does
 
-Armory Spinnaker is a preconfigured distribution of Spinnaker that runs in your Kubernetes cluster. Spinnaker can help you coordinate and orchestrate complicated tasks with pipelines for deploying software.
+## What a Typical Workflow Looks Like
 
 
-## Opinions and Conventions
+A typical work flow with Armory Spinnaker starts with baking a Linux-based machine image. This image along with your launch configurations define an immutable infrastructure that you can use to deploy to your cloud provider with Spinnaker. After the deployment, run your tests, which can be integrated with Spinnaker and automatically triggered. Based on your deployment strategy and any criteria you set, go live with the build.
```

Comment:
there is no difference here between OSS and Armory

Reply:
Removed 'Armory'. I want to build out this section more too at some point and create a high level 'life of a deployment' page or section on this page taking people from commit to my code is live. Let me know if you want to partner on that :)